# To connect MySQL database to Python3

INSTALL PACKAGES

In [ ]:
from __future__ import print_function
import pymysql

CREATE CONNECTION OBJECT

In [13]:
conn = pymysql.connect(host='localhost', port=3306, user='root', passwd='root', db='cooper')

cursor = conn.cursor()

cursor.execute("SELECT count(*) FROM `agent call informaton` where QUARTER = 'Q2' ")

print(cursor.description)

print()

for row in cursor:
    print(row)

cursor.close()
conn.close()



(('count(*)', 8, None, 21, 21, 0, False),)

(1373,)


### Verified Results on MySQL database. The count of records for Agents in Q2 is 1373